core/View/SecurityPolicy.php

```diff
@@ -16,6 +16,12 @@
  */
 class SecurityPolicy
 {
+    /*
+     * Commonly used rules
+     */
+    const RULE_DEFAULT = "'self' 'unsafe-inline' 'unsafe-eval' data:";
```

Comment:
@justinvelluppillai is the `data` needed? I read eg 

> data: Allows data: URIs to be used as a content source. This is insecure; an attacker can also inject arbitrary data: URIs. Use this sparingly and definitely not for scripts.

Reply:
@tsteur Session Recordings need it (TreeMirror js puts a data: image in) which made me expect other places may also. Session Recordings already have it in their rule so I could remove it in RULE_DEFAULT if you think that's unnecessary or better.

Reply:
yeah it be good to have it as much restricted as possible by default. Do we need the `data:` for style attributes? Not sure what it does and fonts?